app/helpers/application_helper.rb

```diff
@@ -64,7 +64,11 @@ def enrichment_summary(summary_list, model, key, value, fields, truncate_value:
       end
     end
   end
-# rubocop:enable Rails/HelperInstanceVariable
+  # rubocop:enable Rails/HelperInstanceVariable
+
+  def display_phase_banner_border?(user)
+    user && !user.admin? && (user.providers_via_organisations.where recruitment_cycle: RecruitmentCycle.current).one?
```

Comment:
```suggestion
user && !user.admin? && (user.providers.where(recruitment_cycle: RecruitmentCycle.current)).one?
```

We've updated to just `users.providers` now and not using the organisation stuff